crates/bls-crypto/src/hash_to_curve/try_and_increment_cip22.rs

```diff
@@ -73,19 +73,32 @@ where
     H: Hasher<Error = BLSError>,
     P: SWModelParameters,
 {
-    /// Hash with attempt takes the input, appends a counter
-    pub fn hash_with_attempt_cip22(
+   /// Hash with attempt takes the input, appends a counter
+   pub fn hash_with_attempt_cip22(
         &self,
         domain: &[u8],
         message: &[u8],
         extra_data: &[u8],
+    ) -> Result<(GroupProjective<P>, usize), BLSError> {
+        let mut counter = 0;
+        self.hash_with_attempt_cip22_counter(domain, message, extra_data, &mut counter)
+    }
+
+ /// Hash with attempt takes the input, appends a counter
+    pub fn hash_with_attempt_cip22_counter(
```

Comment:
I think these aren't needed!
`hash_with_attempt` already returns the counter, as the 2nd element in the tuple.